src/custom/hooks/useApproveCallback/useApproveCallbackMod.ts

```diff
@@ -0,0 +1,124 @@
+import { MaxUint256 } from '@ethersproject/constants'
+import { TransactionResponse } from '@ethersproject/providers'
+import { CurrencyAmount, Percent, Currency, TradeType } from '@uniswap/sdk-core'
+import { Trade as V2Trade } from '@uniswap/v2-sdk'
+import { Trade as V3Trade } from '@uniswap/v3-sdk'
+import { useCallback, useMemo } from 'react'
+import { SWAP_ROUTER_ADDRESSES, V2_ROUTER_ADDRESS } from 'constants/addresses'
+import { useTransactionAdder, useHasPendingApproval } from 'state/enhancedTransactions/hooks'
+import { calculateGasMargin } from 'utils/calculateGasMargin'
+import { useTokenContract } from 'hooks/useContract'
+import { useActiveWeb3React } from 'hooks/web3'
+import { useTokenAllowance } from 'hooks/useTokenAllowance'
+import { HashType } from '@src/custom/state/enhancedTransactions/reducer'
```

Comment:
What is the mod in this file? Only this import?
Also, path

```suggestion
import { HashType } from 'state/enhancedTransactions/reducer'
```